scripts/rvm

```diff
@@ -224,10 +224,10 @@ then
         fi
       elif
         [[ "${__path_to_ruby}" == "${rvm_path}"* ]] &&
-        [[ -z "${GEM_HOME:-}" || -z "${GEM_PATH:-}" ]]
+        [[ -z "${GEM_HOME:-}" ]]
       then
         echo "
-Warning: PATH set to RVM ruby but GEM_HOME and/or GEM_PATH not set, see:
```

Comment:
The fix about GEM_PATH could be valid, but the message would have to be different, as it now points to an issue 3212, which has nothing to do with GEM_PATH.

@mpapis what do you think?

Reply:
Another option could be to warm only on unset GEM_PATH, but to avoid the warning if it's set but empty.

To be honest, I don't fully understand the issue #3212 that is warned about, although I use rvm since many years. There is neither a good description of the impact of the problem nor how it can be solved.

Reply:
I haven't read it in detail. I think it would be enough to remove the reference to the issue and we can merge it in.

Reply:
OK, I removed the reference to the issue, added a CHANGELOG entry and fixed a URI typo in another entry.